src/main/java/mekanism/common/content/gear/mekasuit/ModuleElectrolyticBreathingUnit.java

```diff
@@ -59,8 +59,8 @@ public void tickServer(IModule<ModuleElectrolyticBreathingUnit> module, IModuleC
             productionRate = getMaxRate(module) / 2;
         }
         if (productionRate > 0) {
-            FloatingLong usage = MekanismConfig.general.FROM_H2.get().multiply(2);
-            int maxRate = Math.min(productionRate, module.getContainerEnergy(stack).divideToInt(usage));
+            long usage = Math.multiplyExact(MekanismConfig.general.FROM_H2.get(), 2);
```

Comment:
Similar to another comment, we should just limit the config range rather than throwing an exception at runtime.